src/components/base-address/BaseAddress.vue

```diff
@@ -243,8 +243,8 @@ export default defineComponent({
         localSchema.postalCode = origPostalCodeRules.concat([baseRules.zipCode])
         localSchema.region = origRegionRules
       } else {
-        localSchema.postalCode = origPostalCodeRules.concat([baseRules.maxLength(15)])
-        localSchema.region = [baseRules.maxLength(2), ...spaceRules]
+        localSchema.postalCode = origPostalCodeRules.concat([baseRules[AddressValidationRules.MAX_LENGTH](15)])
```

Comment:
Do you need to import AddressValidationRules ?

Reply:
It's not valid anymore, attempting to fix

Reply:
Is this fixed?